### DISCUSSION TITLE
Is it possible to map `ctrl+.` and `ctrl+,`?

### DISCUSSION THREAD

**Discussion by chanced** [kovidgoyal/kitty#5765] [Q&A] [answered]
I'd really like to be able to map (ctrl + period) and (ctrl + comma) but I can't seem to get the below to work. If it's possible, how can I go about it?
```conf
map ctrl+. send_text all \e[183;5u
map ctrl+, send_text all \e[184;5u
```
thanks!

Operating System: macOS Ventura (v13.0.1)

**Comment by chanced** [comment, marked as answer]
Ah, it has something to do with the escape sequence.

```conf
map ctrl+. send_text all ©+.
map ctrl+, send_text all ©+,
```
works fine

**Comment by 80KiloMett** [comment]
For anyone on Linux looking for an answer to this:
```
map ctrl+. send_text all \u001B[46;5u
map ctrl+, send_text all \u001B[44;5u
```
Tested as keybindings in tmux.
